### PR TITLE
More formatting and polishing

### PR DESCRIPTION
**Pull request by mrakitin**
Here are a few updates to the documentation. I used vim's `gq` command to align the paragraphs of the text, and made a few clarifications and corrections.

### CODE REVIEW

**Comment by dmgav** [comment]
I think everything looks good. No objections on my part.

File: source/creating_packages.rst [hunk 1]
```diff
@@ -115,90 +115,93 @@ If you already have it cloned, pull the latest version of the
...
+- manually create a ``meta.yaml`` recipe file.
 
 
 .. note::
 
-    It is important that each of the following command
-    is run from the correct directory. Since it is easy to lose track of
-    directory changes, all the paths will be specified 
-    relative to the ``$HOME`` directory.
+    It is important that each of the following command is run from the correct
```
Comment (jklynch) [inline source/creating_packages.rst]: command -> commands

File: source/creating_packages.rst [hunk 2]
```diff
@@ -115,90 +115,93 @@ If you already have it cloned, pull the latest version of the
...
     $ mkdir <package-name>
     $ cd <package-name>
 
-The recipe may be created based on ``meta.yaml`` file from
-the respective **conda-forge** feedstock (and sometimes from the original package repository).
-If such file is not available
-or unusable, find an appropriate sample ``meta.yaml`` (from
-a similar package that was successfully built) and modify it.
-Copy ``meta.yaml`` file to the temporary directory you just created:
+The recipe may be created based on ``meta.yaml`` file from the respective
```
Comment (jklynch) [inline source/creating_packages.rst]: ... based on the ``meta.yml`` file ...

File: source/creating_packages.rst (hunk 1 above)
Comment (mrakitin) [reply source/creating_packages.rst]: Done via 0b0d102.

File: source/creating_packages.rst (hunk 2 above)
Comment (mrakitin) [reply source/creating_packages.rst]: Done via 0b0d102.

Comment (mrakitin) [reply source/creating_packages.rst]: Done via 0b0d102.

**Comment by mrakitin** [comment]
@dmgav, @jklynch, thanks for your reviews!

Please note, I had to specify the type for each `code-block`, so that the docs can be build on a fresh install with sphinx v1.8.2. The errors were as follows:
```
19:04 $ make html
Running Sphinx v1.8.2
Adding copy buttons to code blocks...
loading pickled environment... done
[autosummary] generating autosummary for: creating_packages.rst, index.rst
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [] 0 added, 1 changed, 0 removed
reading sources... [100%] creating_packages

Warning, treated as error:
/Users/mrakitin/src/mrakitin/DAMA/nsls-ii-forge/docs/source/creating_packages.rst:341:Error in "code-block" directive:
1 argument(s) required, 0 supplied.

.. code-block::

    * nsls-ii-forge/<package-name>-feedstock has been enabled on azure pipelines
make: *** [html] Error 2
```

**Pull request merged**